_config.yml

```diff
@@ -17,6 +17,9 @@ name: 'Ryan Straits'
 author: 'Ryan Straits'
 author_image: "/assets/images/RMS-large-cntr.jpg"
 
+# plugins
+plugins: ['jekyll-paginate']
```

Comment:
Starting from v3.x.x you must also declare the list of Jekyll plugins to be loaded. It is used to be 'gems' and v3.5.0 has renamed it to 'plugins'.